Assets/Scripts/Presenters/TradePresenter.cs

```diff
@@ -183,6 +204,7 @@ public void SignTradeOffer()
                 if (TradePossibleForOriginator(_currentTrade, _originator))
                 {
                     DecideOnTrade();
+                    GoToNextTribe();
```

Comment:
the ai still need to decide here which could lead to a counteroffer so I don’t think the turn should be over.